pages/mensa.vue

```diff
@@ -13,8 +13,8 @@
       :wrap="$vuetify.breakpoint.smAndDown"
       row>
       <v-flex
-        v-for="dayPlan in weekPlan"
-        :key="dayPlan.date.day()"
+        v-for="dayPlan in $store.state.mensa.weekPlan"
```

Comment:
…das dann wieder in eine computed property stecken? 
Ich weiß nicht, ob `$store` im Template good practice ist.

Reply:
i.o